src/core/server.lua

```diff
@@ -1 +1,9 @@
-_NCS:trace("Loading NCS Core, please wait...")
\ No newline at end of file
+_NCS:trace("Loading NCS Core, please wait...")
+
+---triggerClientEvent
+---@param eventName string
+---@return void
+---@public
+function _NCS:triggerClientEvent(eventName, ...)
+    TriggerClientEvent(self:formatEvent(eventName), ...)
```

Comment:
Missing the target ID, please refer to : 

`TriggerClientEvent(string eventName, int playerId[, ...])`

Reply:
is it correct now?